numpy/linalg/linalg.py

```diff
@@ -930,7 +930,11 @@ def eigvalsh(a, UPLO='L'):
         computed.
     UPLO : {'L', 'U'}, optional
         Same as `lower`, with 'L' for lower and 'U' for upper triangular.
-        Deprecated.
+        Deprecated. 
```

Comment:
@pv, OK. I'll remove it until such time, if ever, it gets harmonized with scipy ;) Speaking of which, we might should do the same for the `qr` decomposition at some point.